ruby/rspec/style_guide.md

```diff
@@ -0,0 +1,645 @@
+# RSpec style guide
+
+This RSpec style guide outlines our recommended best practices so that our developers can write code that can be maintained by other (future) developers.
+This is meant to be a style guide that reflects real-world usage, as well as a
+guide that holds to an ideal that has been agreed upon by many of the people it was intended to be used by.
```

Comment:
Try not to repeat "guide" twice in the same sentence. Aside from that comment, I would re-write the sentence because it is a bit confusing, especially the last part.
Maybe instead of "as well as...", let's try to connect the second part using: "...usage, holding to an ideal..."